practice_app/api/tests.py

```diff
@@ -101,4 +101,37 @@ def test_valid_rows(self):
         self.assertContains(response, 'source')
         self.assertContains(response, 'date')
         self.assertContains(response, 'url')
-        self.assertContains(response, 'position')
\ No newline at end of file
+        self.assertContains(response, 'position')
+class zenodo_test_cases(TestCase):
+
+    def setUp(self):
+        self.c = Client()
+
+    def tearDown(self):
+        print('GET Tests Completed Successfully')
+
+    def test_zenodo_api(self):
+        ACCESS_TOKEN = api_keys.api_keys['zenodo_api']
```

Comment:
After removing Zenodo api test, you won't need to use this access token. In my opinion, it's not a good practice to use access token in unit test.

Reply:
Removed with the Zenodo API call.